include/klee/ExecutionState.h

```diff
@@ -154,17 +153,7 @@ class ExecutionState {
   // The numbers of times this state has run through Executor::stepInstruction
   std::uint64_t steppedInstructions;
 
-private:
-  ExecutionState() : ptreeNode(0) {}
-
-public:
-  ExecutionState(KFunction *kf);
-
-  // XXX total hack, just used to make a state so solver can
-  // use on structure
-  ExecutionState(const std::vector<ref<Expr> > &assumptions);
```

Comment:
I kind of like removing these, but I did find them useful sometimes I think.

Reply:
With the separation you can use references to different constraint sets as it pleases you.
The only use case this function is used is to invoke the solver in the next step.